### PR TITLE
[Fix](executor)Fix alter policy failed

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
Fix alter policy's workload group failed.

```
mysql [(none)]>create workload schedule policy test_cancel_3s_query conditions(query_time > 3000) actions(cancel_query) properties('workload_group'='policy_group');
Query OK, 0 rows affected (0.00 sec)

mysql [(none)]>alter workload schedule policy test_cancel_3s_query properties('workload_group'='');
Query OK, 0 rows affected (0.00 sec)

mysql [(none)]>show workload schedule policy;
+-------+----------------------+-------------------+--------------+----------+---------+---------+---------------+
| Id    | Name                 | Condition         | Action       | Priority | Enabled | Version | WorkloadGroup |
+-------+----------------------+-------------------+--------------+----------+---------+---------+---------------+
| 78073 | test_cancel_3s_query | query_time > 3000 | cancel_query | 0        | true    | 1       | policy_group  |
+-------+----------------------+-------------------+--------------+----------+---------+---------+---------------+
1 row in set (0.00 sec)
```

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadschedpolicy/WorkloadSchedPolicy.java [hunk 1]
```diff
@@ -183,7 +183,11 @@ public void updateProperty(Map<String, String> property, List<Long> wgIdList) {
             this.priority = Integer.parseInt(priorityStr);
         }
 
-        if (wgIdList.size() > 0) {
+        String workloadGroupIdStr = property.get(WORKLOAD_GROUP);
+        // workloadGroupIdStr != null means user set workload group property,
+        // then we should overwrite policy's workloadGroupIdList
+        // if workloadGroupIdStr.length == 0, it means the policy should match all query.
+        if (workloadGroupIdStr != null) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadschedpolicy/WorkloadSchedPolicy.java:190]: using strings.isnullorempty to check the value is null or empty, not only check nullable.

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by yiguolei — DISMISSED** [review]

**Comment by yiguolei** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**